_data/arinerron/arinerron.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Aaron Esau
+institution: Tigard High School
+profile_pic: arinerron2-filled.png	
+quote: i use arch btw '>"><svg/onload=alert()> ${{2*2}}
```

Comment:
Are you sure you want to push this 🤔

Reply:
Alright, alright, updated the quote in https://github.com/Arinerron/graduation/commit/f1b3a24a28de1fdd2165a0435d5340baba24ad08